data/categories.txt

```diff
@@ -2344,7 +2344,7 @@ sw:^Makumbusho|kivutio|utalii
 waterway-waterfall|@tourism
 en:2^Waterfall
 ru:2^Водопад|достопримечательность
-ar:^شلال|مناظر
+ar:^شلال|مناظر|سياحة
```

Comment:
Yes .. the '^' sign point to "شلال" .. that happen because Arabic is Right-to-Left language.
And Yes the two other synonyms needed because if someone search by these other synonyms.
Thanks.